src/android/java/com/pspdfkit/cordova/CordovaPdfActivity.java

```diff
@@ -30,26 +29,26 @@
   private static CordovaPdfActivity currentActivity;
   private final CompositeDisposable compositeDisposable = new CompositeDisposable();
 
+  /** Singleton document listener for dispatching events from the current activity to JavaScript interface. */
   @NonNull
-  private final DocumentListener listener =
-      new SimpleDocumentListener() {
-        @Override
-        public void onDocumentSaved(@NonNull PdfDocument document) {
-          EventDispatcher.getInstance().sendEvent("onDocumentSaved", new JSONObject());
-        }
-
-        @Override
-        public void onDocumentSaveFailed(
-            @NonNull PdfDocument document, @NonNull Throwable exception) {
-          try {
-            final JSONObject data = new JSONObject();
-            data.put("message", exception.getMessage());
-            EventDispatcher.getInstance().sendEvent("onDocumentSaveFailed", data);
-          } catch (JSONException e) {
-            e.printStackTrace();
-          }
-        }
-      };
+  private final static DocumentListener listener = new SimpleDocumentListener() {
+    @Override
+    public void onDocumentSaved(@NonNull PdfDocument document) {
+      EventDispatcher.getInstance().sendEvent("onDocumentSaved", new JSONObject());
+    }
+
+    @Override
+    public void onDocumentSaveFailed(
+        @NonNull PdfDocument document, @NonNull Throwable exception) {
+      try {
+        final JSONObject data = new JSONObject();
+        data.put("message", exception.getMessage());
+        EventDispatcher.getInstance().sendEvent("onDocumentSaveFailed", data);
+      } catch (JSONException e) {
+        e.printStackTrace();
```

Comment:
I realize it was there before, but we probably shouldn't leave `printStackTrace` calls.
Maybe replace it with a proper log statement?

Reply:
Thanks. You're right. I'm going to replace it with a simple exception log for now.